src/Session.php

```diff
@@ -92,6 +92,8 @@ public function open(): void
 
         if ($this->sessionId !== null) {
             session_id($this->sessionId);
+        } else {
+            session_id(bin2hex(random_bytes(16)));
```

Comment:
Why not `session_create_id()`?

Reply:
What about collision check? `session_create_id` also don't check collision if called before start session.

Reply:
session_create_id() is used to create new session id for the current session.
[Reference](https://www.php.net/manual/en/function.session-create-id.php#:~:text=session_create_id()%20is%20used%20to%20create%20new%20session%20id%20for%20the%20current%20session)

Reply:
My bad. I've re-read this doc page, and now I see it's a good solution.